integrations/nvim/queries/tlaplus/highlights.scm

```diff
@@ -219,8 +220,9 @@
 (block_comment_text) @comment
 (comment) @comment
 (single_line) @comment
-(_ label: (identifier) @tag)
-(pcal_goto statement: (identifier) @tag)
+(_ label: (identifier) @label)
```

Comment:
@susliko I changed these to `@label` since it seemed more applicable; thoughts?

Reply:
Looks good